partners/scientific-communities.md

```diff
@@ -52,12 +61,15 @@ Our vibrant community is filled with experts in the Python packaging space. A
 partnership with us means that you will always have support if you run into
```

Comment:
```suggestion
partnership with pyOpenSci means that you will always have support if you run into
```

Reply:
can we be more specific on what "support" looks like? maybe something like "always have access to support from our community of Python experts" etc.

Reply:
great feedback! i've modified it to this:


> Our vibrant community is filled with experts in the Python packaging space. A
> partnership with us means that you will always have support from our diverse community of Pythonistas if you run into
> Python packaging and software maintenance challenges.